.husky/post-merge

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx fresko
```

Comment:
We could add a comment here to explain that using `npx` instead of `yarn` will allow users to download the bin if they do not have it yet.

Reply:
That's very true! I can add a comment, no worries 😉

Reply:
I agree, we can merge with the comment

Reply:
@adriencaccia I just updated the code to include the comment, it also includes a little fix that I encountered with non-interactive terminal